pkg/machinery/resources/runtime/kernel_params_spec.go

```diff
@@ -43,6 +46,18 @@ func NewKernelParamSpec(namespace resource.Namespace, id resource.ID) *KernelPar
 	return r
 }
 
+// NewKernelParamDefaultSpec initializes a KernelParamSpec resource.
+func NewKernelParamDefaultSpec(namespace resource.Namespace, id resource.ID) *KernelParamSpec {
```

Comment:
this is a bit hacky, but it won't let us see the resource with `talosctl get`, as ResourceDefinition mentions only one type of the two

Reply:
yeah... I know. But I couldn't find any other options so far.
We can also use a different namespace or prefix resource ids with something, but that also sounds hacky.